modules/imgproc/src/filter.simd.hpp

```diff
@@ -295,6 +295,7 @@ int FilterEngine__proceed(FilterEngine& this_, const uchar* src, int srcstep, in
     return dy;
 }
 
+CV_NO_UBSAN
```

Comment:
It is better to avoid disabling of whole function.

Adjust single expression only through `#if` check.

Reply:
Finally, suppression should not be necessary at all.